src/components/textInput/TextInput.stories.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import TextInput from './TextInput';
+import configureStore from '../../redux/store';
+import { Provider } from 'react-redux';
+
+const TextInputWithProvider = () => (
+  <Provider store={configureStore()}>
+    <TextInput name="Example" value="" onChange={() => {}} label="Example" />
+  </Provider>
+);
+
+export default {
+  title: 'TextInput',
+  component: TextInput,
+};
+
+export const NormalTexInput = () => <TextInputWithProvider />;
```

Comment:
Typo here, missing letter T in NormalTextInput

Reply:
Done